packages/server/lib/plugins/child/run_require_async_child.js

```diff
@@ -78,69 +78,55 @@ function run (ipc, file, projectRoot) {
     return false
   }
 
-  ipc.on('loadLegacyPlugins', async (legacyConfig) => {
-    try {
-      let legacyPlugins = require(file)
-
-      if (legacyPlugins && typeof legacyPlugins.default === 'function') {
-        legacyPlugins = legacyPlugins.default
-      }
+  // Config file loading of modules is tested within
```

Comment:
This got butchered during the rebase, I put all the "legacy config" stuff at the bottom.